### PR TITLE
Landing Pages Tests

### PR DESCRIPTION
**Pull request by virlatinus**
| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? | N/A
| Deprecations? | N/A

#### Description:
This PR creates a set of unit tests for the PageBundle. 

#### Steps to test this PR:
1. Apply this PR
2. Run `composer test`. All tests should pass.
3. Create or Edit a Landing Page, everything should work as before.

### CODE REVIEW

File: app/bundles/PageBundle/Tests/EventListener/PageSubscriberTest.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            ->getMock();
+
+        $pageBuilderEvent = new PageBuilderEvent($translator);
+        $pageBuilderEvent->addToken('{token_test}', 'TOKEN VALUE');
+        $tokens = $pageBuilderEvent->getTokens();
+
+        foreach ($tokens as $token) {
+            if ('{token_test}' === $token) {
+                $this->assertEquals($tokens[$token], 'TOKEN VALUE');
+                $this->assertArrayHasKey('{token_test}', $tokens);
```
Comment (escopecz) [inline app/bundles/PageBundle/Tests/EventListener/PageSubscriberTest.php]: This assertion should be probably outside of the foreach loop, right? It's enough to assert that once.

File: app/bundles/PageBundle/Tests/Model/PageModelTest.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+use Mautic\PageBundle\Entity\Page;
+use Mautic\PageBundle\Tests\PageTestAbstract;
+
+class PageModelTest extends PageTestAbstract
+{
+    public function testCreate()
+    {
+        $page      = new Page();
+        $pageModel = $this->getPageModel();
```
Comment (escopecz) [inline app/bundles/PageBundle/Tests/Model/PageModelTest.php]: This is not testing anything. Shouldn't we remove it?

File: app/bundles/PageBundle/Tests/PageTestAbstract.php [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+            ->method('generateRedirectUrl')
+            ->willReturn('http://some-url.com');
+
+        return $mockRedirectModel;
+    }
+
+    /**
+     * @return VideoModel
+     */
+    protected function getVideoModel()
```
Comment (escopecz) [inline app/bundles/PageBundle/Tests/PageTestAbstract.php]: This isn't used anywhere. What's the reason for it?

**Comment by dongilbert** [comment]
@virlatinus Can you review the requested changes here?

**Comment by virlatinus** [comment]
@escopecz, I fixed the issues you pointed out. Thanks

**Pull request merged**